a.go

```diff
@@ -141,7 +142,7 @@ func (a *A) FailNow() {
 // Skipped reports whether the task was skipped.
 func (a *A) Skipped() bool {
 	a.mu.Lock()
-	res := a.skipped
+	res := *a.skipped
```

Comment:
Try calling Skipped() without calling Skip - I think it will panic, then next step you'll probably want to conditionalize it and it will even work - but it's not thread safe because the mutex is locked on the copy, not on the main object. That's why I used functions. Failed too

Reply:
I do not think so as this is a pointer to the original mutex but I will double check